var/spack/repos/builtin/packages/hsa-rocr-dev/package.py

```diff
@@ -14,20 +14,29 @@ class HsaRocrDev(CMakePackage):
        Linux HSA Runtime for Boltzmann (ROCm) platforms."""
 
     homepage = "https://github.com/RadeonOpenCompute/ROCR-Runtime"
-    url      = "https://github.com/RadeonOpenCompute/ROCR-Runtime/archive/rocm-3.5.0.tar.gz"
+    url      = "https://github.com/RadeonOpenCompute/ROCR-Runtime/archive/rocm-3.7.0.tar.gz"
 
     maintainers = ['srekolam', 'arjun-raj-kuppala']
 
+    version('3.7.0', sha256='0071d14431f73ce74574e61d0786f2b7cf34b14ea898a1f54b6e1b06b2d468c0')
     version('3.5.0', sha256='52c12eec3e3404c0749c70f156229786ee0c3e6d3c979aed9bbaea500fa1f3b8')
 
     variant('build_type', default='Release', values=("Release", "Debug"), description='CMake build type')
 
-    for ver in ['3.5.0']:
+    depends_on('cmake@3:', type="build")
+    for ver in ['3.5.0', '3.7.0']:
         depends_on('hsakmt-roct@' + ver, type=('link', 'run'), when='@' + ver)
+        depends_on('llvm-amdgpu@' + ver, type=('link', 'run'), when='@' + ver)
         depends_on('libelf@0.8:', type='link', when="@" + ver)
```

Comment:
for rocm-3.5, hsa-rocr-dev is not dependent on llvm-amdgpu  . So, would be good if you can put on something like t def add_deps_for_version(self, version):
        if version >= Version('3.7.0'):
            depends_on('llvm-amdgpu@3.7.0' , type=('build','run'), when="@3.7.0")